regulation/changes.py

```diff
@@ -18,6 +18,8 @@
 
 logger = logging.getLogger(__name__)
 
+TAGS_WITH_SUBCONTENT = ["{eregs}part", "{eregs}subpart"]
+TAGS_WITH_NO_SUBCONTENT = ["{eregs}title"]
```

Comment:
:+1: You keep finding these things we've taken for granted and improving them. I love it.